_stbt/imgproc_cache.py

```diff
@@ -352,7 +352,7 @@ def match_all():
     cached_time, uncached_time, cached_result, uncached_result = (
         _check_cache_behaviour(match_all))
 
-    assert uncached_time > (cached_time * 4)
+    assert uncached_time > (cached_time * 2)
```

Comment:
Presumably you had to change this because it got faster,

Reply:
Yeah, when it failed the uncached time was 0.0148 and cached time was 0.0038, so 1 extra millisecond in unchached time would have passed.